pages/dkp/konvoy/1.8/storage/static-local-volume/index.md

```diff
@@ -8,65 +8,49 @@ beta: false
 enterprise: false
 ---
 
-<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
-
 The `localvolumeprovisioner` addon uses the [local volume static provisioner][localstorage] to manage persistent volumes for pre-allocated disks.
 It does this by watching the `/mnt/disks` folder on each host and creating persistent volumes in the `localvolumeprovisioner` storage class for each disk that is discovered in this folder.
 
-- Persistent volumes with a 'Filesystem' volume-mode are discovered if they are mounted under `/mnt/disks`.
+-   Persistent volumes with a 'Filesystem' volume-mode are discovered if they are mounted under `/mnt/disks`.
 
-- Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
+-   Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
 
-# Before you begin
+## Before you begin
 
 Before starting this tutorial, you should verify the following:
 
-- You have access to a Linux, macOS, or Windows computer with a supported operating system version.
-
-- You have a provisioned `konvoy` cluster that is configured to use the `localvolumeprovisioner` addon, but have not any addons to the cluster.
+-   You have access to a Linux, macOS, or Windows computer with a supported operating system version.
 
-  For this tutorial, you **should not deploy** using all of the default settings as described in the [Quick start][quickstart].
+-   You have a provisioned `konvoy` cluster that is configured to use the `localvolumeprovisioner` addon, but have not any addons to the cluster.
```

Comment:
I have seen this kind of wording before, and I believe there is a verb or something missing in the second phrase. Or am I missing something? 
```suggestion
-   You have a provisioned `konvoy` cluster that is configured to use the `localvolumeprovisioner` addon, but have not added any addons to the cluster.
```